.github/workflows/ci.yml

```diff
@@ -3,13 +3,46 @@ name: Node CI
 on: [push, pull_request]
 
 jobs:
+  validate-yarn-cache:
```

Comment:
Nit: maybe this should be called `prepare-yarn-cache`, since "validate" implies that it would fail the build if, for example, the cache is old.

Reply:
I have added a `"name"` section in https://github.com/babel/babel/pull/12002/files/d484dbba30163e84d5b8cc610efea652b50c6a67..303ec4a3229d17092d770e01cbe6a77bcc72a075#diff-e9f950f17198d3d5e3122a44230a09b9R7

I can port the name changes here and also rename the job id.